examples/README.md

```diff
@@ -0,0 +1,21 @@
+# Purpose of the examples
+
+This directory contains various examples of SLOs defined with OpenSLO with additional context. You can use it as a supplement for spec to
```

Comment:
Should we say "SLOs compliant to the OpenSLO specification with additional context"?